### DISCUSSION TITLE
Provided Edge Function is too large

### DISCUSSION THREAD

**Discussion by BrickheadJohnny** [vercel/community#1079] [Help] [answered]
Hi there, our team started experiencing a strange error with our deployments (Next.js project deployed on Vercel) today. We're getting a "Provided Edge Function is too large" error, and I'm pretty sure it comes from our link preview image generator endpoints, since we only use edge functions for that.

When I build our project, I can see that basically all of our serverless functions are 698kB, which is way above the 500kB limit, but not sure why we get this error now and why we haven't seen it before?

It's also really strange that all of our serverless functions are the same size, even the simplest ones?

<img width="641" alt="image" src="https://user-images.githubusercontent.com/92519134/203356338-c155d29d-973c-4789-a93b-c1116ae63ed5.png">

**Comment by amyegan** [comment, marked as answer]
Sorry you're running into this error, @BrickheadJohnny. I know very little about our Edge architecture, so I'm reaching out to the experts for some help. I'll let you know when I learn more

**Reply by amyegan** [reply]
The team just release a fix for this today. If you redeploy then you should get more information about the specific edge function that is causing the problem. 🙏

**Reply by BrickheadJohnny** [reply]
@amyegan thank you so much for the quick response and solution, it's working now!
